starters/nextjs-starter-ts/components/article-view.tsx

```diff
@@ -26,7 +31,14 @@ export default function ArticleView({
     () => data?.article ?? article,
     [data, article],
   );
-  const articleTitle = useArticleTitle(hydratedArticle);
+
+  return (
+    <StaticArticleView article={hydratedArticle} onlyContent={onlyContent} />
+  );
+}
+
+export function StaticArticleView({ article, onlyContent }: ArticleViewProps) {
```

Comment:
Would appreciate suggestions on names for this, calling it `StaticArticleView` and the component with realtime hydration just `ArticleView` doesn't seem right

Reply:
I'm okay with this name actually. Alternate ideas:
HydratingArticleView, ArticleView
ArticleViewWithUpdates, ArticleView

Or something like this:
<HydrateArticle renderer={ArticleView} />

Not as nice, but a HOC kind of approach, but it seems worse. ArticleView would have to make article optional, since we are expecting HydrateArticle to provide it when it renders its children.
<HydrateArticle article={article}>
  <ArticleView onlyContent={onlyContent}>
</HydrateArticle>